Website/Components/Post/Post.razor.css

```
@@ -1,24 +1,31 @@
 .card {
-  /* border: 1px solid; */
-  border-left: 0.5rem solid;
+  border-left: 0.4rem solid;
   border-color: grey;
   padding: 0.5rem;
   border-radius: 0;
+  background-color: #1b2042;
+}
+.card:hover {
+  background-color: red;
 }
 .icon-1x {
   font-size: 24px !important;
+  color: white;
 }
 a {
   text-decoration: none;
 }
+span {
+  color: white;
+}
 .text-primary,
 a.text-primary:focus,
 a.text-primary:hover {
-  color: #00adbb !important;
+  color: white !important;
```

```suggestion
  color: #00adbb !important;
```
I like having the link differentiate itself from the details better